### ISSUE TITLE
[Bug]: xz-utils: Bad system call

### ISSUE THREAD

**Issue by tgies** [termux/termux-packages#19347]
### Problem description

Since the latest update to `liblzma` and `xz-utils`, `xz` is dying with `SIGSYS` (bad system call).

Attaching full `strace`, but this looks like the relevant bit:
```
landlock_create_ruleset(NULL, 0, LANDLOCK_CREATE_RULESET_VERSION) = 0
--- SIGSYS {si_signo=SIGSYS, si_code=SYS_SECCOMP, si_call_addr=0x7bcfc63240, si_syscall=__NR_landlock_create_ruleset, si_arch=AUDIT_ARCH_AARCH64} ---
```
[xz-strace.txt](https://github.com/termux/termux-packages/files/14397852/xz-strace.txt)
[pkg-info-liblzma.txt](https://github.com/termux/termux-packages/files/14397851/pkg-info-liblzma.txt)
[pkg-info-xz-utils.txt](https://github.com/termux/termux-packages/files/14397853/pkg-info-xz-utils.txt)

### What steps will reproduce the bug?

```
$ xz
```

### System information

termux-info:

```
Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=26980
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
TERMUX__USER_ID=0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://mirror.mwt.me/termux/main stable main
# tur-repo (sources.list.d/tur.list)
deb https://tur.kcubeterm.com tur-packages tur tur-on-device tur-continuous
# x11-repo (sources.list.d/x11.list)
deb https://mirror.mwt.me/termux/x11 x11 main
Updatable packages:
All packages up to date
termux-tools version:
1.40.6
Android version:
14
Kernel build information:
Linux localhost 5.15.110-android14-11-gcc48824eebe8-ab10865596 #1 SMP PREEMPT Tue Sep 26 19:57:58 UTC 2023 aarch64 Android
Device manufacturer:
Google
Device model:
Pixel 8 Pro
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
Installed termux plugins:
com.termux.widget versionCode:13
com.termux.api versionCode:51
com.termux.styling versionCode:31
```

**Comment by tgies** [comment]
I now see this appears to be fixed by the pending PR #19346, which I had missed due to the default `is:issue` filter.

**Comment by TomJo2000** [comment]
Closing this as completed, if the issue reoccurs either comment in this issue or open a new one.
